packages/app/documents/service.ts

```diff
@@ -369,6 +371,84 @@ export async function cloneDocument(
     }
 }
 
+/**
+ * apply a set of JSON patch operations to a list of documents
+ */
+export async function bulkPatchDocuments(
+    documentTitles: Array<string>,
+    modelName: string,
+    user: User,
+    operations: JSONPatchDocument
+) {
+    try {
+        const result = await Promise.allSettled(
```

Comment:
This may need some work...right now, we're looping over each document, which ends up being a bit slow. Trying to make use of existing service methods like "getDocument" and "createDocument"

Going to open a different story to look at performance improvements